### PR TITLE
4863 - Fix list not showing on accordion after typing a character not available on the list

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes a bug where accordion list will not show again if the character typed on autocomplete input is not available in the words on the list.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/4863

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/applicationmenu/example-personalized.html
- Click inside the search field box and type in "zzzzzz" 
- Remove the word "zzzz" and clear the search.

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
- [x] A note to the change log.

### CODE REVIEW

**Comment by ericangeles** [comment]
@tjamesallen15 looks like a couple of pages affect this fix

https://github.com/infor-design/enterprise/pull/5359/checks?check_run_id=2903244916

**Review by ericangeles — CHANGES_REQUESTED** [review]
@tjamesallen15 looks like a couple of pages were affected by this fix.

https://github.com/infor-design/enterprise/pull/5359/checks?check_run_id=2903244916

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I see a related bug now. If you type zzz and hit the x the x button no longer is also clearing the app menu

**Comment by tmcconechy** [comment]
@tjamesallen15 i found one more case that is not working. Maybe we need another or different key event (keypress or input?)

- go in the field and type zzzzzz
- select all text and press cmd+x (cut)

File: src/components/applicationmenu/applicationmenu.js [hunk 1]
```diff
@@ -236,6 +236,24 @@ ApplicationMenu.prototype = {
     $(document).on('keydown.applicationmenu', (e) => {
       self.handleKeyDown(e);
     });
+
+    $(document).on('input.applicationmenu', () => {
```
Comment (tmcconechy) [inline src/components/applicationmenu/applicationmenu.js:240]: I think we need to add a tear down to https://github.com/infor-design/enterprise/blob/main/src/components/applicationmenu/applicationmenu.js#L717 maybe?

Comment (tjamesallen15) [reply src/components/applicationmenu/applicationmenu.js:240]: Hello @tmcconechy , I added the teardown right under line 741. Did I put it wrong?

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**